app/src/main/java/nerd/tuxmobil/fahrplan/congress/models/ScheduleData.kt

```diff
@@ -18,4 +18,8 @@ data class ScheduleData(
                 .flatMap { it.lectures.asSequence() }
                 .firstOrNull { it.lectureId == lectureId }
     }
+
+    fun findRoomIndex(lecture: Lecture): Int {
```

Comment:
The ScheduleData class ist not unit tested at all. Could you please add unit tests?

Reply:
@ideadapt @cketti I now wrote unit tests for the class and added the commit to the top of the branch. See 4a430f995235af0a1a41e1dd792a18bf82e1558b. I also added KDoc to `ScheduleData`, see 3ff7372c5a88f934bf30f73a932e0c73fd02e042. Please let me know if you spot any aspect I missed in the unit tests.